#Aquest fitxer serveix per deixar els problemes que no arreglem pero que es tenen que mirar

	-En l'apartat de templates index.html es te que mirar com agafar totes les taules en comptes de com ho he ficat jo a ma

	-Tenim que modificar els templates de com mostrar-ho, sol esta fet una petita previsualitzaci√≥



ERROR CLIENTDETAIL

AttributeError at /app/Client/3/

type object 'Client' has no attribute '_default_manager'

Request Method: 	GET
Request URL: 	http://127.0.0.1:8000/app/Client/3/
Django Version: 	1.7.4
Exception Type: 	AttributeError
Exception Value: 	

type object 'Client' has no attribute '_default_manager'

Exception Location: 	/usr/local/lib/python2.7/dist-packages/django/views/generic/detail.py in get_queryset, line 67
Python Executable: 	/usr/bin/python
Python Version: 	2.7.3
Python Path: 	

['/home/eloi/SITW/django_supermarkets/supermarket',
 '/usr/lib/python2.7',
 '/usr/lib/python2.7/plat-linux2',
 '/usr/lib/python2.7/lib-tk',
 '/usr/lib/python2.7/lib-old',
 '/usr/lib/python2.7/lib-dynload',
 '/usr/local/lib/python2.7/dist-packages',
 '/usr/lib/python2.7/dist-packages',
 '/usr/lib/python2.7/dist-packages/PIL',
 '/usr/lib/python2.7/dist-packages/gst-0.10',
 '/usr/lib/python2.7/dist-packages/gtk-2.0',
 '/usr/lib/pymodules/python2.7',
 '/usr/lib/python2.7/dist-packages/wx-2.8-gtk2-unicode']

Server time: 	Fri, 1 May 2015 19:35:52 +0000


Environment:



Request Method: GET
Request URL: http://127.0.0.1:8000/app/Client/3/




Traceback:
File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py" in get_response
  111.                     response = wrapped_callback(request, *callback_args, **callback_kwargs)
File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py" in view
  69.             return self.dispatch(request, *args, **kwargs)
File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py" in dispatch
  87.         return handler(request, *args, **kwargs)
File "/usr/local/lib/python2.7/dist-packages/django/views/generic/detail.py" in get
  114.         self.object = self.get_object()
File "/usr/local/lib/python2.7/dist-packages/django/views/generic/detail.py" in get_object
  31.             queryset = self.get_queryset()
File "/usr/local/lib/python2.7/dist-packages/django/views/generic/detail.py" in get_queryset
  67.                 return self.model._default_manager.all()

Exception Type: AttributeError at /app/Client/3/
Exception Value: type object 'Client' has no attribute '_default_manager'

